component-models.json

```diff
@@ -1287,27 +1335,100 @@
       {
         "component": "aem-content",
         "valueType": "string",
-        "name": "link",
+        "name": "card_link",
         "label": "Product Page Link"
       },
       {
         "component": "richtext",
         "valueType": "string",
-        "name": "key-feature_amount",
-        "label": "Amount Key Feature"
+        "name": "keyfeatures_heading",
```

Comment:
this should not be a `richtext` but a text. You use `richtext` if the author can create 1..n paragraphs of different types. for the heading you would probably want to use a `text` field together with a `keyfeatures_headingType` field.